biahub/cli/slurm.py

```diff
@@ -0,0 +1,95 @@
+import subprocess
+import time
+
+from tqdm import tqdm
+
+
+def wait_for_jobs_to_finish(
+    job_ids: list[str],
+    sleep_time: int = 60,
+    use_sacct: bool = False,
+) -> None:
```

Comment:
Doesn't seem to work for local.

Reply:
There's an executor method (`.wait()`) that does this: https://github.com/facebookincubator/submitit/blob/64119dc669a21d69f46c9d9a3f556ce447d238d3/submitit/local/local.py#L343

Reply:
More examples using `job.result()`: https://github.com/facebookincubator/submitit/blob/main/docs/examples.md